### ISSUE TITLE
Error: looks like we got no XML document

### ISSUE THREAD

**Issue by gjelden** [jamesiarmes/php-ews#223]
I've got this error message ("looks like we got no XML document") because the content of one email had the character "&#x19 ;".

I've fixed this problem by adding the folowing line:
    $response = preg_replace('/&#x[0-1]?[0-9A-E];/', ' ', $response);
to NTLMSoapClient.php after line 82 (after the line with curl_exec)

**Comment by Longcat00** [comment]
So the regex pasted here doesn't cover all possibilities since hex goes to F.

Proper:
```
$response = preg_replace('/&#x[0-1]?[0-9A-E];/', ' ', $response);
```

**Comment by darpla** [comment]
Symbols 0x9, 0xA and 0xD are allowed, so here is my version of the quick fix:
$response = preg_replace('/(?!&#x0?(9|A|D))(&#x[0-1]?[0-9A-F];)/', ' ', $response);

**Comment by jamesiarmes** [comment]
I'd love to resolve this issue, and I've left a comment on the pull request as well, but I need to be able to reproduce to ensure this works. If somebody could provide me with an example request I could look into this further.

It sounds like this issue would require an email one of these characters to already exist in the user's mailbox. If somebody has an email they could forward to me, please reach out to me and I will give you an email address you could forward it to.

**Comment by jamesiarmes** [comment]
I found a way to reproduce this issue and have committed this fix in jamesiarmes/php-ntlm@b6c3ee5ba980fdf5e8e7803a8b8a6f0de874b731. Running `composer update` should get you the latest code from this repo.
